packages/lesswrong/components/async/editor-plugins/linkifyPlugin.js

```diff
@@ -167,23 +168,26 @@ const createLinkifyPlugin = () => {
             }),
           });
         } else {
+          // LESSWRONG: This branch was causing a bunch of problems and is pretty buggy, so I deactivated it
+          
           // We are no longer in a URL but the entity is still present. Remove it from
           // the current character so the linkifying "ends".
-          const entityRange = new SelectionState({
```

Comment:
I'd say the main thing that needs testing here is all the conditions in which you're typing a linkified link, and then aren't anymore. Eg: you type `http://www.example.com<space>`, or you put your cursor in the middle and type `http://www.exa<space>mple.com`.